store/store.go

```diff
@@ -0,0 +1,96 @@
+/*
+Package store contains functions for writing parsed feeds to cloud storage.
+*/
+package store
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/spaolacci/murmur3"
+)
+
+// AWSCredentials stores the region and bucket needed to access an S3 bucket.
+type AWSCredentials struct {
+	Region string
+	Bucket string
+}
+
+// A Document is the generic representation of an individual entry in a feed
+type Document struct {
+	FeedID       string
+	LanguageCode string
+	Year         int
+	Month        string
+	Day          int
+	Title        string
+	Description  string
+	Content      string
```

Comment:
In my previous comment, I mentioned that for the encoding task we could set this as `[]byte` and from the implementation here it looks like that might be the way to go. Additionally, we could add an `Encoding` field to this struct to determine how to decode the string if we were going to reuse this struct for parsing.